setup.md

```diff
@@ -344,7 +344,7 @@ included in the [repository](https://github.com/carpentries-incubator/template).
 
 6.  At this point, you should have a page like this:
 
-    ![]({{ page.root }}/fig/using-github-import.png)
+    ![Screenshot of GitHub pageview that displays after following the steps described for importing your project into GitHub]({{ page.root }}/fig/using-github-import.png)
```

Comment:
I wonder if, in this context, longer alt-text is needed considering that we are using the image for the user to verify that they have the correct page (and yes, the image is woefully out of date) and users who rely on screenreaders to navigate the page will not have the same information that users who rely on vision do. 

```suggestion
    ![Screenshot of web form that says "Import your project to GitHub" showing two fields with "Your old repositoy's clone URL" set to "carpentries/styles" and "Your new repository details" set to "timtomch" for the owner and "data-cleanup" for the name]({{ page.root }}/fig/using-github-import.png)
```